src/sentry/api/endpoints/chunk_upload.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import absolute_import
+
+# from rest_framework.permissions import IsAuthenticated
+from rest_framework import status
+from rest_framework.response import Response
+
+from sentry.models import FileBlob
+from sentry.models.file import DEFAULT_BLOB_SIZE
+from sentry.api.base import Endpoint
+from sentry.api.bases.project import ProjectReleasePermission
+
+
+UPLOAD_ENDPOINT = 'https://sentry.io'
+MAX_CHUNKS_PER_REQUEST = 16
+MAX_CONCURRENCY = 4
```

Comment:
Is this just a hint to the person that calls this? Because there's nothing that's actually enforcing this.

Reply:
Yeah, it should be a hint, @mitsuhiko mentioned something about if we have a different endpoint which handles connections differently we can change it afterward.